### PR TITLE
Update FAQ.

### PR DESCRIPTION
**Pull request by justinxzhao**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -1,89 +1,92 @@
...
 
-### Main architect and maintainer
+Decoder-only, encoder-only, encoder-decoder, vanilla feed-forward, transformers,
+and more have all been implemented in Ludwig.
 
-[Piero Molino](http://w4nderlu.st) (Stanford University, previously at Uber AI) is the creator, main architect and maintainer of Ludwig.
+# What does Ludwig not support (yet)?
 
-### Co-maintainers
+Domains of deep learning that Ludwig does not support:
```
Comment (w4nderlust) [inline docs/faq.md:59]: (yet)

File: docs/faq.md [hunk 2]
```diff
@@ -1,89 +1,92 @@
...
-  author = {Piero Molino and Yaroslav Dudin and Sai Sumanth Miryala},
-  title = {Ludwig: a type-based declarative deep learning toolbox},
-  year = {2019},
-  eprint = {arXiv:1909.07930},
-}
-```
+Ludwig was created in 2019 by Piero Molino, Yaroslav Dudin, and Sai Sumanth
+Miryala while at Uber AI.
+
+Today, Ludwig is open source, licensed by the Linux Foundation, with source code
```
Comment (w4nderlust) [inline docs/faq.md:85]: licensed -> supported

File: docs/faq.md [hunk 3]
```diff
@@ -1,89 +1,92 @@
...
 
-```
-@misc{Molino2019,
-  author = {Piero Molino and Yaroslav Dudin and Sai Sumanth Miryala},
-  title = {Ludwig: a type-based declarative deep learning toolbox},
-  year = {2019},
-  eprint = {arXiv:1909.07930},
-}
-```
+Ludwig was created in 2019 by Piero Molino, Yaroslav Dudin, and Sai Sumanth
```
Comment (w4nderlust) [inline docs/faq.md:82]: by Piero Molino, with help from

Comment (justinxzhao) [reply docs/faq.md:82]: Done.

**Review by w4nderlust — APPROVED** [review]

**Pull request merged**